NuGet dependencies conflict

error on NuGet package:
```
CSC : error CS1703: Multiple assemblies with equivalent identity have been imported: 'C:\Users\巍\.nuget\packages\FChan.Library\1.0.7\lib\portable-net45+wp80+win+MonoAndroid10+xamarinios10+MonoTouch10\System.Net.Http.dll' and 'C:\Users\巍\.nuget\packages\System.Net.Http\4.0.0\ref\netcore50\System.Net.Http.dll'. Remove one of the duplicate references.
```

--- a/FChan.Library.Test/ChanTest.cs
+++ b/FChan.Library.Test/ChanTest.cs
@@ -12,7 +12,7 @@ namespace FChan.Library.Test
             try
             {
                 BoardRootObject board = Chan.GetBoard();
-				Assert.NotNull(board);
+                Assert.NotNull(board);
             }
             catch(Exception e)
             {
@@ -26,7 +26,7 @@ namespace FChan.Library.Test
             try
             {
                 BoardRootObject board = await Chan.GetBoardAsync();
-				Assert.NotNull(board);
+                Assert.NotNull(board);
             }
             catch(Exception e)
             {
@@ -40,7 +40,7 @@ namespace FChan.Library.Test
             try
             {
                 ThreadRootObject thread = Chan.GetThreadPage("a",1);
-				Assert.NotNull(thread);
+                Assert.NotNull(thread);
             }
             catch(Exception e)
             {
@@ -62,112 +62,112 @@ namespace FChan.Library.Test
             }
         }
 
-		[Test()]
-		public async void GetAllBoardsAsync()
-		{
-			BoardRootObject boardRoot = Chan.GetBoard();
-			Assert.NotNull(boardRoot);
+        [Test()]
+        public async void GetAllBoardsAsync()
+        {
+            BoardRootObject boardRoot = Chan.GetBoard();
+            Assert.NotNull(boardRoot);
 
-			foreach (var board in boardRoot.Boards) 
-			{
-				for (int i = 1; i <= board.Pages; i++)
-				{
-					ThreadRootObject thread = await Chan.GetThreadPageAsync(board.BoardName, i);
+            foreach (var board in boardRoot.Boards) 
+            {
+                for (int i = 1; i <= board.Pages; i++)
+                {
+                    ThreadRootObject thread = await Chan.GetThreadPageAsync(board.BoardName, i);
 
-					Assert.NotNull(thread);
+                    Assert.NotNull(thread);
 
-					foreach (var item in thread.Threads) 
-					{
-						Assert.NotNull(item);
+                    foreach (var item in thread.Threads) 
+                    {
+                        Assert.NotNull(item);
 
-						foreach (var post in item.Posts) 
-						{
-							Assert.NotNull(post);
-							Assert.AreNotEqual(post.PostNumber, 0);
-						}
-					}
-				}
-			}
-		}
+                        foreach (var post in item.Posts) 
+                        {
+                            Assert.NotNull(post);
+                            Assert.AreNotEqual(post.PostNumber, 0);
+                        }
+                    }
+                }
+            }
+        }
 
-		[Test()]
-		public async void GetAllThreadAsync()
-		{
-			try
-			{
-				ThreadRootObject thread = await Chan.GetThreadPageAsync("a",1);
+        [Test()]
+        public async void GetAllThreadAsync()
+        {
+            try
+            {
+                ThreadRootObject thread = await Chan.GetThreadPageAsync("a",1);
 
-				Assert.NotNull(thread);
+                Assert.NotNull(thread);
 
-				foreach (var item in thread.Threads) 
-				{
-					Assert.NotNull(item);
+                foreach (var item in thread.Threads) 
+                {
+                    Assert.NotNull(item);
 
-					foreach (var post in item.Posts) 
-					{
-						Assert.NotNull(post);
-						Assert.AreNotEqual(post.PostNumber, 0);
-					}
-				}
-			}
-			catch(Exception e)
-			{
-				Assert.Fail(e.Message);
-			}
-		}
+                    foreach (var post in item.Posts) 
+                    {
+                        Assert.NotNull(post);
+                        Assert.AreNotEqual(post.PostNumber, 0);
+                    }
+                }
+            }
+            catch(Exception e)
+            {
+                Assert.Fail(e.Message);
+            }
+        }
 
-		[Test()]
-		public void GetAllBoards()
-		{
-			BoardRootObject boardRoot = Chan.GetBoard();
-			Assert.NotNull(boardRoot);
+        [Test()]
+        public void GetAllBoards()
+        {
+            BoardRootObject boardRoot = Chan.GetBoard();
+            Assert.NotNull(boardRoot);
 
-			foreach (var board in boardRoot.Boards) 
-			{
-				for (int i = 1; i <= board.Pages; i++)
-				{
-					ThreadRootObject thread = Chan.GetThreadPage(board.BoardName, i);
+            foreach (var board in boardRoot.Boards) 
+            {
+                for (int i = 1; i <= board.Pages; i++)
+                {
+                    ThreadRootObject thread = Chan.GetThreadPage(board.BoardName, i);
 
-					Assert.NotNull(thread);
+                    Assert.NotNull(thread);
 
-					foreach (var item in thread.Threads) 
-					{
-						Assert.NotNull(item);
+                    foreach (var item in thread.Threads) 
+                    {
+                        Assert.NotNull(item);
 
-						foreach (var post in item.Posts) 
-						{
-							Assert.NotNull(post);
-							Assert.AreNotEqual(post.PostNumber, 0);
-						}
-					}
-				}
-			}
-		}
+                        foreach (var post in item.Posts) 
+                        {
+                            Assert.NotNull(post);
+                            Assert.AreNotEqual(post.PostNumber, 0);
+                        }
+                    }
+                }
+            }
+        }
 
-		[Test()]
-		public void GetAllThread()
-		{
-			try
-			{
-				ThreadRootObject thread = Chan.GetThreadPage("a",1);
+        [Test()]
+        public void GetAllThread()
+        {
+            try
+            {
+                ThreadRootObject thread = Chan.GetThreadPage("a",1);
 
-				Assert.NotNull(thread);
+                Assert.NotNull(thread);
 
-				foreach (var item in thread.Threads) 
-				{
-					Assert.NotNull(item);
+                foreach (var item in thread.Threads) 
+                {
+                    Assert.NotNull(item);
 
-					foreach (var post in item.Posts) 
-					{
-						Assert.NotNull(post);
-						Assert.AreNotEqual(post.PostNumber, 0);
-					}
-				}
-			}
-			catch(Exception e)
-			{
-				Assert.Fail(e.Message);
-			}
-		}
+                    foreach (var post in item.Posts) 
+                    {
+                        Assert.NotNull(post);
+                        Assert.AreNotEqual(post.PostNumber, 0);
+                    }
+                }
+            }
+            catch(Exception e)
+            {
+                Assert.Fail(e.Message);
+            }
+        }
     }
 }

--- a/FChan.Library/Chan.cs
+++ b/FChan.Library/Chan.cs
@@ -1,30 +1,31 @@
-﻿using System;
-using System.Net;
+﻿using Newtonsoft.Json;
+using System;
 using System.IO;
-using Newtonsoft.Json;
-using System.Threading.Tasks;
+using System.Net;
 using System.Text;
-using System.Net.Http;
-using System.Linq;
+using System.Threading.Tasks;
 
 namespace FChan.Library
 {
-	/// <summary>
-	/// Chan.
-	/// </summary>
-	public static class Chan
-	{
+    /// <summary>
+    /// Chan.
+    /// </summary>
+    public static class Chan
+    {
         internal static T DownloadObject<T>(string url)
+        {
+            var task = DownloadObjectAsync<T>(url);
+            task.Wait();
+            return task.Result;
+        }
+
+        internal static async Task<T> DownloadObjectAsync<T>(string url)
         {
             try
             {
-                using (var client = new HttpClient())
-                {
-                    Task<string> response = client.GetStringAsync(url);
-                    response.Wait();
-                    string responseString = response.Result;
-                    return JsonConvert.DeserializeObject<T>(responseString);
-                }
+                string response = await GetStringAsync(url);
+                string responseString = response;
+                return JsonConvert.DeserializeObject<T>(responseString);
             }
             catch
             {
@@ -32,20 +33,33 @@ namespace FChan.Library
             }
         }
 
-        internal static async Task<T> DownloadObjectAsync<T>(string url)
+        public static async Task<string> GetStringAsync(string url)
         {
-            try
+            var request = WebRequest.CreateHttp(url);
+            request.Method = "GET";
+
+            var task = new TaskCompletionSource<WebResponse>();
+
+            request.BeginGetResponse(ac =>
             {
-                using (var client = new HttpClient())
+                try
                 {
-                    string response = await client.GetStringAsync(url);
-                    string responseString = response;
-                    return JsonConvert.DeserializeObject<T>(responseString);
+                    task.SetResult(request.EndGetResponse(ac));
                 }
-            }
-            catch
+                catch (Exception e)
+                {
+                    task.SetException(e);
+                }
+            }, null);
+
+            using (var response = await task.Task)
+            using (var stream = response.GetResponseStream())
+            using (var output = new MemoryStream())
             {
-                return default(T);
+
+                await stream.CopyToAsync(output);
+                var array = output.ToArray();
+                return Encoding.UTF8.GetString(array, 0, array.Length);
             }
         }
 
@@ -78,7 +92,7 @@ namespace FChan.Library
             ThreadRootObject thread = DownloadObject<ThreadRootObject>(Constants.GetThreadPageUrl(board, page));
 
             foreach (Thread item in thread.Threads)
-                foreach (Post post in item.Posts) 
+                foreach (Post post in item.Posts)
                     post.Board = board;
 
             return thread;
@@ -95,7 +109,7 @@ namespace FChan.Library
             ThreadRootObject thread = await DownloadObjectAsync<ThreadRootObject>(Constants.GetThreadPageUrl(board, page));
 
             foreach (Thread item in thread.Threads)
-                foreach (Post post in item.Posts) 
+                foreach (Post post in item.Posts)
                     post.Board = board;
 
             return thread;
@@ -130,5 +144,5 @@ namespace FChan.Library
 
             return thread;
         }
-	}
+    }
 }

--- a/FChan.Library/FChan.Library.csproj
+++ b/FChan.Library/FChan.Library.csproj
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
@@ -10,6 +10,8 @@
     <AssemblyName>FChan.Library</AssemblyName>
     <TargetFrameworkProfile>Profile78</TargetFrameworkProfile>
     <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
+    <NuGetPackageImportStamp>
+    </NuGetPackageImportStamp>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
@@ -47,25 +49,13 @@
     <Compile Include="Constants.cs" />
   </ItemGroup>
   <Import Project="$(MSBuildExtensionsPath32)\Microsoft\Portable\$(TargetFrameworkVersion)\Microsoft.Portable.CSharp.targets" />
-  <Import Project="..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets" Condition="Exists('..\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets')" />
-  <ItemGroup>
-    <Folder Include="Model\" />
-  </ItemGroup>
-  <ItemGroup>
-    <Reference Include="System.Net.Http">
-      <HintPath>..\packages\Microsoft.Net.Http.2.2.29\lib\portable-net40+sl4+win8+wp71+wpa81\System.Net.Http.dll</HintPath>
-    </Reference>
-    <Reference Include="System.Net.Http.Primitives">
-      <HintPath>..\packages\Microsoft.Net.Http.2.2.29\lib\portable-net40+sl4+win8+wp71+wpa81\System.Net.Http.Primitives.dll</HintPath>
-    </Reference>
-    <Reference Include="System.Net.Http.Extensions">
-      <HintPath>..\packages\Microsoft.Net.Http.2.2.29\lib\portable-net40+sl4+win8+wp71+wpa81\System.Net.Http.Extensions.dll</HintPath>
-    </Reference>
-    <Reference Include="Newtonsoft.Json">
-      <HintPath>..\packages\Newtonsoft.Json.8.0.2\lib\portable-net45+wp80+win8+wpa81+dnxcore50\Newtonsoft.Json.dll</HintPath>
-    </Reference>
-  </ItemGroup>
   <ItemGroup>
     <None Include="packages.config" />
+  </ItemGroup>
+  <ItemGroup>
+    <Reference Include="Newtonsoft.Json, Version=8.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <HintPath>..\packages\Newtonsoft.Json.8.0.1\lib\portable-net40+sl5+wp80+win8+wpa81\Newtonsoft.Json.dll</HintPath>
+      <Private>True</Private>
+    </Reference>
   </ItemGroup>
 </Project>

--- a/FChan.Library/packages.config
+++ b/FChan.Library/packages.config
@@ -1,7 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Microsoft.Bcl" version="1.1.10" targetFramework="portable-net45+win+wp80+MonoTouch10+MonoAndroid10+xamarinmac20+xamarinios10" />
-  <package id="Microsoft.Bcl.Build" version="1.0.21" targetFramework="portable-net45+win+wp80+MonoTouch10+MonoAndroid10+xamarinmac20+xamarinios10" />
-  <package id="Microsoft.Net.Http" version="2.2.29" targetFramework="portable-net45+win+wp80+MonoTouch10+MonoAndroid10+xamarinmac20+xamarinios10" />
-  <package id="Newtonsoft.Json" version="8.0.2" targetFramework="portable-net45+win+wp80+MonoAndroid10+xamarinios10+MonoTouch10" />
+  <package id="Newtonsoft.Json" version="8.0.1" targetFramework="portable45-net45+win8+wp8" />
 </packages>

